src/components/form/fields/phoneNumberField.tsx

```diff
@@ -86,6 +86,8 @@ const PhoneNumberField = (props: PhoneNumberFieldProps) => {
     } = props;
 
     const [labels, setLabels] = useState<Labels>()
+    const currentValue = value !== null && typeof value === 'object' && 'raw' in value ? value.raw : value
```

Comment:
Je sais que c'était déjà comme ça mais ça doit être un oublie. J'avais écris une fonction utilitaire pour simplifier ça :
```suggestion
    const currentValue = isRichFormValue(value, 'raw') ? value.raw : value
```
From 
```typescript
import { isRichFormValue } from '../../../helpers/utils';
```